src/mame/drivers/midqslvr.cpp

```diff
@@ -2,25 +2,254 @@
+|                U11   U7        U8                                                          |
+|                                                                                            |
+|                                                                                            |
+|                                 U6                                                         |
+|JP11           Y2    U20                                                                    |
+|                                                                                            |
+|          U25                                    U4                                         |
+|                                                                      S2        S1          |
+|JP16      U15   U16   U17    U19   U18    U5     U13    U12    U14                          |
+|                                                                                            |
+|                                                                                            |
+|                                                                                            |
+|      JP17         JP19           JP18              JP13             JP14           JP15    |
+|--------------------------------------------------------------------------------------------|
+
+Notes:
+    J1/JP5: Video connector from/to video card, not used
+    J2: USB port, not used
+    JP1-4: Video signal jumpers, not used
+    JP6: EGA resolution video port, not used
+    JP9: Power connector
+    JP10: 20 pin ribbon cable connector to wheel driver board
+    JP11: 20 pin ribbon cable connector, not used
+    JP12-14: connectors, not used
+    JP15: 14 pin connector for analog controls/potentiometers
+    JP16: 15 pin connector, attack button
+    JP17: connector, not used
+    JP18: 14 pin connector for coin, service, volume and test inputs
+    JP19: 8 pin connector, start button
+    JP20: connector for development use, not used
+    JP21: 2 pin connector for watchdog reset
+    JP22: 9 pin connector for coin meter
+    JP23: Alternate RS232 port
+    JP24: connector, not used
+    P1: DB9 RS-232 port to computer
+	S1: Dip Switches (8)
+	S1: Dip Switches (8)
+         S1-7: *Off: UART, On: USB
+         S1-8: Off: Watchdog Disabled, *On: Watchdog Enabled
+	S2: Dip Switches (8)
+    U4-5: MC74HC273A Octal D Flip-Flop (LS273 based)
+    U6: Not known yet
+    U7/U11: Atmel 24C01A Serial EEPROM
+    U8: MAX707CSA Supervisory Circuit
+    U12-19: HC541 Octal Buffer
+    U20: Philips P87C51/2 8-bit Microcontroller
+    U21: DS14185WM RS-232 Interface IC
+	U22-24: ULN2064B Darlington Transistor
+    U25: Not known yet
+    Y2: FS14.74 Crystal/Oscilator
+
+AT LEAST I'LL GET MY WASHING DONE
+
+Misc Info (HT and OT):
+Both I/O boards could functionwise be very similar. Both video cards are CPLD protected and has their signal converted to EGA resolution.
```

Comment:
Please re-phrase as "Both I/O boards appear similar, hardware-wise."

"could functionwise be very similar" is meaningless conjecture.

Reply:
Deleted the entire paragraph because I noticed I said the same thing. Redundant info mistake.